src/string-utilities.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2020  Matthew "strager" Glazar
+// See end of file for extended copyright information.
+
+#include <climits>
+#include <cstdio>
+#include <cstdlib>
+#include <quick-lint-js/string-utilities.h>
+
+namespace quick_lint_js {
+// Memory management in C and auto allocating sprintf() - asprintf():
+// https://insanecoding.blogspot.com/2014/06/memory-management-in-c-and-auto.html
```

Comment:
Must fix: What is the license for this code? I suspect we are not complying with the license.

(I think we don't need this code anyway. See my comment in c-api.cpp. But if we do need this code, we should comply with the author's license.)